druid/Dockerfile

```
@@ -32,8 +32,26 @@ RUN curl -L https://repo.stackable.tech/repository/packages/druid/apache-druid-$
     # Install the Prometheus emitter extension. This bundle contains the emitter and all jar dependencies.
     curl https://repo.stackable.tech/repository/packages/druid/druid-prometheus-emitter-${PRODUCT}.tar.gz | tar -xzC /stackable/druid/extensions
 
-# Mitigate CVE-2021-44228 (Log4Shell)
+# ===
+# Mitigation for CVE-2021-44228 (Log4Shell)
+# This variable is supported as of Log4j version 2.10 and
+# disables the vulnerable feature
 ENV LOG4J_FORMAT_MSG_NO_LOOKUPS=true
 
+# For earlier versions this script removes the .class file that contains the
+# vulnerable code.
+# TODO: This can be restricted to target only versions which do not honor the environment
+#   varible that has been set above but this has not currently been implemented
+COPY shared/log4shell.sh /bin
+RUN /bin/log4shell.sh /stackable/druid-${PRODUCT}
```

There is no `/stackable/druid-${PRODUCT}` folder. Remove the `${PRODUCT}` suffix.

```suggestion
RUN /bin/log4shell.sh /stackable/druid
```

---

I've added the 'apache-' prefix, I'd rather not rely on the symlink, just to exclude any potentially weird behavior of the script when ran with symlinks.

---

Sorry I didn't mention but the path needs to be updated in all Dockerfiles.

---

I've gone over the files once more, I only found mismatches for Hive & Spark though..

---

I see this when I try to build the Zookeeper image:

---

Looks like the zookeeper path should be `/stackable/apache-zookeeper-${PRODUCT}-bin` rather than `/stackable/zookeeper-${PRODUCT}`